src/themes/default/components/core/AddToCart.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <button-full @click.native="addToCart(product)" :disabled="canBeAdded(product)" data-testid="addToCart">
+  <button-full @click.native="addToCart(product)" :disabled="cantBeAdded(product)" data-testid="addToCart">
```

Comment:
```suggestion
  <button-full @click.native="addToCart(product)" :disabled="isDisabled" data-testid="addToCart">
```
this will be a result of computed property